packages/cogify/src/cogify/cli/cli.cover.ts

```diff
@@ -62,14 +73,25 @@ export const BasemapsCogifyCoverCommand = command({
       defaultValue: () => false,
       defaultValueIsSerializable: true,
     }),
+    name: option({
+      type: optional(string),
+      long: 'name',
+      description: 'Define the name of the output imagery',
+    }),
+    background: option({
+      type: optional(string),
```

Comment:
we should parse this as a command parser so we get earlier errors for invalid colors.